client/common/src/util/markdown/markdown.ts

```diff
@@ -162,6 +162,60 @@ export const renderMarkdown = (
                     'flex-basis': ALL_VALUES_ALLOWED,
                 },
             },
+            transformTags: {
+                object(tagName, attribs) {
+                    if (!attribs.data.includes('base64,')) {
+                        return {
+                            tagName,
+                            attribs,
+                        }
+                    }
+
+                    // Anchor tags can be used to invalidate base64 and bypass filters
+                    const b64object = attribs.data.split('base64,')[1].split('#')[0]
+                    let cleaned
+
+                    try {
+                        cleaned = sanitize(window.atob(b64object), {
+                            allowedTags: sanitize.defaults.allowedTags.concat([
+                                'svg',
+                                'xmlns',
+                                'path',
+                                'picture',
+                                'circle',
+                                'pattern',
+                                'rect',
+                            ]),
+                            allowedAttributes: {
+                                svg: ['width', 'height', 'viewbox', 'version', 'preserveaspectratio', 'style'],
+                                rect: ['x', 'y', 'width', 'height', 'transform', ...svgPresentationAttributes],
+                                path: ['d', ...svgPresentationAttributes],
+                                circle: ['cx', 'cy', ...svgPresentationAttributes],
+                            },
+                            allowedStyles: {
+                                svg: {
+                                    flex: ALL_VALUES_ALLOWED,
+                                    'flex-grow': ALL_VALUES_ALLOWED,
+                                    'flex-shrink': ALL_VALUES_ALLOWED,
+                                    'flex-basis': ALL_VALUES_ALLOWED,
+                                },
+                            },
+                        })
+                    } catch (any) {
```

Comment:
```suggestion
                    } catch (error) {
```

@evict the linter is upset with `any`